graphql/util.go

```diff
@@ -241,3 +241,22 @@ type PatchVariantsTasksRequest struct {
 	Tasks         []string             `json:"tasks"`                    // old format
 	Description   string               `json:"description"`
 }
+
+// BuildFromGqlInput takes a PatchReconfigure gql type and returns a PatchVariantsTasksRequest type
+func (p *PatchVariantsTasksRequest) BuildFromGqlInput(r PatchReconfigure) *PatchVariantsTasksRequest {
+	p.Description = *r.Description
```

Comment:
This field either needs to be non-nullable or you need to handle it being null here before dereferencing

Reply:
changed the description field in schema to be non-nullable